iocore/net/QUICPacketHandler.cc

```diff
@@ -59,7 +59,8 @@ QUICPacketHandler::_read_connection_id(IOBufferBlock *block)
 //
 // QUICPacketHandlerIn
 //
-QUICPacketHandlerIn::QUICPacketHandlerIn(const NetProcessor::AcceptOptions &opt, SSL_CTX *ctx) : NetAccept(opt), _ssl_ctx(ctx)
+QUICPacketHandlerIn::QUICPacketHandlerIn(const NetProcessor::AcceptOptions &opt, SSL_CTX *ctx)
+  : NetAccept(opt), _ctable(65521), _ssl_ctx(ctx)
```

Comment:
any reason to choose this number?
maybe make it configurable?

Reply:
65521 is primer number. I try to make the hash table size a prime number.
I think, to define a default when create the table.  When the load factor > 4, the table will be resized.
OK, I add a configuration item for ConnectionTable.